### PR TITLE
add IndexSz field to TableInfo, to show it through badger info tool

### PR DESCRIPTION
**Pull request by NamanJain8**
Badger info currently does not show the size of table index. 
This PR adds the table index size to badger info and adds a new field `IndexSz` to TableInfo struct. This field denotes the size of the length of the table's block Indices.

This PR addresses DGRAPH-2298.

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/badger/1469)

### CODE REVIEW

File: table/table.go [hunk 1]
```diff
@@ -647,6 +647,9 @@ func (t *Table) blockOffsetsCacheKey() []byte {
 	return buf
 }
 
+// IndexSize size is the length of block indices
+func (t *Table) IndexSize() int { return len(t.blockIndex) }
```
Comment (jarifibrahim) [inline table/table.go:651]: len(t.blockIndex) isn't the actual size of the table index. The actual size can be found as
```go
	indexSz := 0

	for _, bi := range t.blockOffsets() {
		indexSz += bi.Size()
	}
	return indexSz
```

Use the `blockOffsets()` function and not `t.blockIndex`. `t.blockIndex` can be nil if indices are cached.

**Review by jarifibrahim — CHANGES_REQUESTED** [review]
Got a comment.

File: table/table.go (hunk 1 above)
Comment (NamanJain8) [reply table/table.go]: Thanks. Done.

**Review by jarifibrahim — APPROVED** [review]

File: table/table.go [hunk 2]
```diff
@@ -647,6 +647,15 @@ func (t *Table) blockOffsetsCacheKey() []byte {
 	return buf
 }
 
+// IndexSize size is the length of block indices
```
Comment (jarifibrahim) [inline table/table.go:650]: Fix this comment.

**Comment by jarifibrahim** [comment]
@NamanJain8 This PR should be cherry-picked to `release/v2.2007` branch as well.

File: table/table.go (hunk 2 above)
Comment (NamanJain8) [reply table/table.go]: done

**Pull request merged**